config/plugins.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+/**
+ * This file contains a list of plugins and their install paths.
+ *
+ * This is primarily used for identifying plugins in /vendor
+ */
+return [
```

Comment:
I agree, returning is more natural. We could get rid of `$config` all together?  Shouldn't be a problem to check for it and show an error to developers so that they can update their existing files.